### ISSUE TITLE
AssertionError when running tests with babel

### ISSUE THREAD

**Issue by labrys** [hgrecco/pint#663]
FAIL: test_babel (test_babel.TestBabel)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "e:\src\pint\pint\testsuite\test_babel.py", line 19, in test_babel
    "24.0 m�tres"
AssertionError: '24.0 {0} m�tres' != '24.0 m�tres'
- 24.0 {0} m�tres
?     ----
+ 24.0 m�tres

**Comment by jondoesntgit** [comment]
I get the extra {0} error as well. I'm working on a Mac.

**Comment by karelyatin** [comment]
I am also facing the issue, it's reproducable with tox.
# install tox and git
git clone https://github.com/hgrecco/pint
cd pint
sed -i '/deps = coverage/a \       babel' tox.ini
sed -i 's/pint setup.py test/pint setup.py test -s pint.testsuite.test_babel/' tox.ini
tox -epy27

**Comment by bors-ltd** [comment]
Hi,

I ended up here searching for this bug too, trying the tutorial:
```
In [8]: ureg['1.3 * meter / (second**2)'].format_babel(locale='fr_FR')                                   
Out[8]: '1.3 {0} mètre par {0} seconde²'
```
It happens in any language:
```
In [10]: ureg['1.3 * meter / (second**2)'].format_babel(locale='it_IT')                                  
Out[10]: '1.3 {0} metri al {0} secondi²'

In [11]: ureg['1.3 * meter / (second**2)'].format_babel(locale='en_US')                                  
Out[11]: '1.3 {0} meters per {0} seconds²'
```
Maybe I can contribute a fix but I was expecting to find the translations somewhere in this project.

**Comment by bors-ltd** [comment]
I tried downgrading Babel back to version 2.2.0, even in that old version, the root locale data file was already using this format:
```
ipdb> data['unit_patterns']['electric-ampere:short']                                                                                                                                  
{'other': '{0} A'}
```

Could it simply be this search&replace commit that broke it? https://github.com/hgrecco/pint/commit/546ddaae991fcae400db22f87e82fffdaf5b2316#diff-fe8a9337c84de477ae6fbd590aefd9efR152

I opened the PR #773 to revert it.

But it still makes no sense the build is passing, according to the home page.
